lib/instrumentation/modules/http.js

```diff
@@ -24,7 +25,7 @@ module.exports = function (http, agent, version, enabled) {
 
       var result = original.apply(this, arguments)
 
-      var trans = agent._instrumentation.currentTransaction
+      var trans = this[symbols.requestTransaction]
```

Comment:
Would it make sense to set `this[symbols.requestTransaction] = null` afterwards? I can't currently think of any leak related to this, but when ever we set something on an object that's not under our control, there's always a risk of something going bad - so better safe than sorry right?

Reply:
done.